packages/kotti-ui/source/kotti-field/tests.ts

```diff
@@ -106,7 +70,7 @@ describe('useField', () => {
 	it('props.value gets deepCloned', async () => {
 		const VALUE_REFERENCE = { something: 'something' }
 
-		const wrapper = shallowMount(TestComponentObject, {
```

Comment:
This is wrong, since this test needs props.value to be something worth deep-cloning.

Reply:
the value reference is still deep cloned, the only difference between `TestComponentObject` and `TestComponent` was the `isCorrectDataType` check